src/main/java/emu/grasscutter/game/managers/deforestation/DeforestationManager.java

```diff
@@ -35,7 +35,12 @@ public class DeforestationManager extends BasePlayerManager {
         ColliderTypeToWoodItemID.put(9,101309);
         ColliderTypeToWoodItemID.put(10,101310);
         ColliderTypeToWoodItemID.put(11,101311);
-        ColliderTypeToWoodItemID.put(12,101312);
+        ColliderTypeToWoodItemID.put(12,101312);        
```

Comment:
```suggestion
        ColliderTypeToWoodItemID.put(12,101312);
```
<img width="536" alt="image" src="https://user-images.githubusercontent.com/5397662/202891683-11256ce5-bd04-41ad-ad9f-5a5d17262318.png">